### PR TITLE
Introduce `source.wipe` to storage-pool config

### PR DESCRIPTION
**Pull request by stgraber**

### CODE REVIEW

**Comment by stgraber** [comment]
@masnax I've confirmed that you can add this to a MemberConfig even if the existing cluster does not expose the key and it will work to wipe the device:
```
cluster.MemberConfig = append(cluster.MemberConfig, api.ClusterMemberConfigKey{
    Entity: "storage-pool",
    Name:   "local",
    Key:    "source.wipe",
    Value:  "true",
})
```

File: lxd/storage/drivers/utils.go [hunk 1]
```diff
@@ -836,3 +837,30 @@ func loopDeviceAutoDetach(loopDevPath string) error {
 	_, err := shared.RunCommand("losetup", "--detach", loopDevPath)
 	return err
 }
+
+// wipeBlockHeaders will wipe the first 4MB of a block device.
```
Comment (tomponline) [inline lxd/storage/drivers/utils.go:841]: Why 4mb out of interest?

**Review by tomponline — CHANGES_REQUESTED** [review]
Please can you include the disk path in the errors "Failed to wipe headers from disk: "

Thanks

File: lxd/storage/drivers/utils.go (hunk 1 above)
Comment (stgraber) [reply lxd/storage/drivers/utils.go:841]: No real reason, it's been my habit for years.
You normally would need a lot less, but wiping a few MBs tends to help with the few tools that scan for more than one superblock on the likes of ext

Comment (tomponline) [reply lxd/storage/drivers/utils.go:841]: Thanks.

**Comment by stgraber** [comment]
@tomponline done

**Review by tomponline — APPROVED** [review]

**Comment by stgraber** [comment]
@tomponline good to go

**Pull request merged**